addon/components/paper-dialog-inner.js

```diff
@@ -4,10 +4,10 @@ import PaperDialog from './paper-dialog';
 
 const { Component, computed } = Ember;
 
-export default Component.extend(Translate3dMixin, {
+export default Component.extend(/*Translate3dMixin, TODO */ {
   tagName: 'md-dialog',
-  classNames: ['md-default-theme'],
-  classNameBindings: ['contentOverflow:md-content-overflow', 'fullscreen:md-dialog-fullscreen'],
+  classNames: ['md-default-theme'     , 'md-transition-in'     /* TODO */ ],
+  classNameBindings: ['contentOverflow:md-content-overflow', 'dialogComponent.fullscreen:md-dialog-fullscreen'],
```

Comment:
That discussion does not apply for private components.
We had to resort to such mechanisms because we can't expect the user to pass them by us.